docs/vgclog_examples.md

```diff
@@ -706,3 +707,895 @@ Analyzing the structure and elements of this example log output shows that this
+
+...
+
+<gc-start type="global mark phase"/>      (final STW GMP subincrement starts.)
+
+...   
+<gc-end type="global mark phase"/>        (final STW GMP subincrement ends. No concurrent subincrement runs)
+
+<cycle-end type="global mark phase"/>     (end of global mark cycle)
+
+<exclusive-end/>                          (STW pause ends)    
+
+<exclusive-start/>                        (STW pause starts)
+
+<cycle-start type="partial gc" />         (partial cycle starts)
+
+...
+
+<gc-op type="sweep" />                    (Sweep operation associated with global mark cycle runs)
+
+...
+
+<cycle-end type="partial gc"/>            (partial GC cycle ends)
+
+<exclusive-end/>                          (STw pause ends)
+
+```
+
+
+#### Global mark phase
+
+The first activity of the global mark cycle is a STW pause, recorded by an `<exclusive-start>` element. The GC pauses application threads to execute the cycle's initial operations.
+
+```xml
+<exclusive-start id="1152" timestamp="2021-02-26T11:17:25.033" intervalms="1931.263">
+  <response-info timems="3.082" idlems="1.041" threads="2" lastid="00000000006EDE00" lastname="RunDataWriter.1" />
+</exclusive-start>
+```
+
+The cycle is triggered when the allocation taxation threshold is reached. The <allocation-taxation> element records the memory threshold value that was in use when this example global mark cycle was triggered. The`taxation-threshold` attribute shows the taxation threshold is 1024MB (1,073,741,824), which is half the total memory of the *eden* space (2048 MB), as expected for threshold triggers of global mark cycles and increments. For more information taxation thresholds for the `balanced` policy, see [`balanced` examples](vgclog_examples.md#balanced-examples).
+
+```xml
+<allocation-taxation id="1153" taxation-threshold="1073741824" timestamp="2021-02-26T11:17:25.034" intervalms="1931.251" />
+```
+
+Details about the start of the global mark GC cycle are recorded by the `<cycle-start>` element. The cycle is recorded as type `global mark phase` with an `id=1154`. Any subsequent elements associated with this cycle have a `contextid=1154` to match the global mark GC cycle's `id`. You can use the `contextid` value to distinguish increments and operations of the global mark GC cycle from the partial cycles that interleave with it.
+
+```xml
+<cycle-start id="1154" type="global mark phase" contextid="0" timestamp="2021-02-26T11:17:25.034" intervalms="374365.075" />
+```
+
+The cycle begins with the STW subincrement of a global *mark* phase increment. The STW subincrement is recorded using the `<gc-start>` element of type `global mark phase`.
+
+```xml
+<gc-start id="1155" type="global mark phase" contextid="1154" timestamp="2021-02-26T11:17:25.034">
+  <mem-info id="1156" free="1442840576" total="4294967296" percent="33">
+    <remembered-set count="2197888" freebytes="162912768" totalbytes="171704320" percent="94" regionsoverflowed="3" regionsstable="130" regionsrebuilding="0"/>
```

Comment:
this does not be a sample genuinely from first GMP increment. they would typically have regionsoverflowed = 0, and regionsrebuilding non-0 (the count being exactly as count of overflowed ones that we observed from previous PGC end)

GMP will try to rebuild some or all of overflowed regions, which as GMP progresses may again overflow, or eventually be successfully rebuilt.